benchmark/metrics/test/test_cputime.py

```diff
@@ -204,7 +213,8 @@ def filter(node_cpu_times):
             for process_times in node_cpu_times.values():
                 if process_times:
                     for process in process_times:
-                        process_times[process] = 0
+                        if process != WALLCLOCK_LABEL:
+                            process_times[process] = 0
```

Comment:
:question: If you're bored, it might be fun to try expressing this w/ pmap.transform (http://pyrsistent.readthedocs.org/en/latest/intro.html#transformations)